From 1bb962dc84b76958c0b827812323569c714474e4 Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Sat, 1 Mar 2014 07:42:11 +0100
Subject: [PATCH 1/4] Stop passing static start and stop priorities to
 update-rc.d.

Those are determined dynamically nowadays via the dependencies
specified in the LSB header.
---
 debian/changelog | 9 +++++++++
 debian/rules     | 1 -
 2 files changed, 9 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index 7fb1545..ef391c7 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,12 @@
+bluez (4.101-4.1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * Stop passing static start and stop priorities to update-rc.d. Those are
+    determined dynamically nowadays via the dependencies specified in the
+    LSB header.
+
+ -- Michael Biebl <biebl@debian.org>  Sat, 01 Mar 2014 07:36:31 +0100
+
 bluez (4.101-4) unstable; urgency=low
 
   * Remove 0008-Enable-the-Gateway-and-Source-audio-profiles-by-defa.patch.
diff --git a/debian/rules b/debian/rules
index 7963b09..5023628 100755
--- a/debian/rules
+++ b/debian/rules
@@ -7,7 +7,6 @@ include /usr/share/cdbs/1/rules/patchsys-quilt.mk
 
 LDFLAGS += -Wl,--as-needed
 
-DEB_UPDATE_RCD_PARAMS = "start 25 2 3 4 5 . stop 74 0 1 6 ."
 DEB_DH_INSTALLINIT_ARGS = "--name=bluetooth"
 DEB_CONFIGURE_EXTRA_FLAGS := \
                              --libdir=\$${prefix}/lib/$(DEB_HOST_MULTIARCH) \
-- 
1.9.0

